src/qml/PluginManagerSettings.qml

```diff
@@ -190,6 +192,48 @@ Popup {
     }
   }
 
+  Dialog {
+    id: authorDetails
+    title: authorName
+    parent: mainWindow.contentItem
+    x: ( mainWindow.width - width ) / 2
+    y: ( mainWindow.height - height - 80 ) / 2
+
+    property string authorName: ""
+    property string authorHomepage: ""
+
+    Column {
+      width: childrenRect.width
+      height: childrenRect.height
+      spacing: 10
+
+      TextMetrics {
+        id: authorWarningLabelMetrics
+        font: authorWarningLabel.font
+        text: authorWarningLabel.text
+      }
+
+      Label {
+        id: authorWarningLabel
+        width: mainWindow.width - 60 < authorWarningLabelMetrics.width ? mainWindow.width - 60 : authorWarningLabelMetrics.width
+        text: qsTr("⚠ The author(s) name is self-reported by the plugin and is not independently verified. Please make sure you trust the plugin's origin.")
+        wrapMode: Text.WordWrap
+        font: Theme.defaultFont
+        color: Theme.mainTextColor
+      }
+    }
+
+    standardButtons: authorHomepage === "" ? Dialog.Close : Dialog.Ok | Dialog.Close
+    onAccepted: {
+      Qt.openUrlExternally(authorHomepage)
+    }
+    onAboutToShow: {
+      if (authorHomepage !== "") {
+        standardButton(Dialog.Ok).text = qsTr("Visit author's homepage")
```

Comment:
Do we have the url visible (I think a simple link `<a href="https://whitehouse.gov">https://whitehouse.gov</a>` would be most transparent to the user)